walker/walk.go

```diff
@@ -26,6 +26,11 @@ func isIgnored(filePath string) bool {
 		if utils.StringInSlice(path, ignoreDirs) {
 			return true
 		}
+		if skipDirectories != nil {
+			if utils.StringInSlice(path, skipDirectories) {
+				return true
+			}
+		}
```

Comment:
Even if `skipDirectories` is nil, `StringInSlice` just returns false.
https://play.golang.org/p/ujs5JjC8Vul

So, it looks like we don't need the nil check.

```suggestion
		if utils.StringInSlice(path, skipDirectories) {
				return true
		}
```